projects/admin/src/app/circulation/circulation.module.ts

```diff
@@ -46,6 +46,12 @@ import { RecordModule } from '@rero/ng-core';
 import { RequestedComponent } from './patron/requested/requested.component';
 import { RequestedItemComponent } from './patron/requested/requested-item/requested-item.component';
 import { RequestedItemsListComponent } from './requested-items-list/requested-items-list.component';
+import {
+  OverdueTransactionComponent
+} from './patron/patron-transactions/patron-transaction/overdue-transaction/overdue-transaction.component';
+import {
+  DefaultTransactionComponent
+} from './patron/patron-transactions/patron-transaction/default-transaction/default-transaction.component';
```

Comment:
Can you sort imports? If you are using VSCodium, there is a command to perform this (Organize Imports)